force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +452,17 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        // Check if person1 is older than person2
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        } 
+        // Check if person2 is older than person1
+        else if (person2.getAge() > person1.getAge()) {
+            return person2;
+        } 
+        // If they are the same age, return either person
+        else {
+            return person1; // or return person2;
+        }
     }
```

Comment:
Good but I don't think you need that that extra if else. If your first condition you could check >=  and return person1

else return person 2